docs/components/DocsSideNavigation.js

```diff
@@ -56,9 +56,13 @@ export default function DocsSideNavigation({ border }: {| border?: boolean |}):
             >
               {section.pages.map((componentName, i) => {
                 const href = `/${componentName.replace(/ /g, '_').replace(/'/g, '').toLowerCase()}`;
+
+                // in nextjs, if it's a dynamic route, the dynamic route id will be passed as part of the query obj
+                const { id: pathId } = router.query;
+
                 return (
                   <SideNavigation.TopItem
-                    active={router.pathname === href ? 'page' : undefined}
+                    active={router.pathname === href || `/${pathId}` === href ? 'page' : undefined}
```

Comment:
let's make sure this is still valid after Ayesha's navigation implementation.

Reply:
Good catch. I see some merge conflicts now, so I'll take a look!